cmsplugin_cascade/icon/mixins.py

```diff
@@ -38,7 +39,7 @@ def get_form(self, request, obj=None, **kwargs):
         return super().get_form(request, obj, **kwargs)
 
     def render(self, context, instance, placeholder):
-        context = self.super(IconPluginMixin, self).render(context, instance, placeholder)
+        context = super(IconPluginMixin, self).render(context, instance, placeholder)
```

Comment:
`self.super()`is required so that strides work, because there another base-class is used.

Reply:
if I leave this (self), stride does not work with icon plugins.

Reply:
OK. Then I need to have a look myself.